osprey_coordinator/src/proto/mod.rs

```diff
@@ -1,66 +1,50 @@
 #![allow(unused_imports)]
 
-pub mod discord_smite_rpc {
-    pub mod actions {
-        pub mod v1 {
-            include!(concat!(env!("OUT_DIR"), "/discord_smite_rpc.actions.v1.rs"));
-        }
-    }
-
-    pub mod common {
-        pub mod v1 {
-            include!(concat!(env!("OUT_DIR"), "/discord_smite_rpc.common.v1.rs"));
-        }
-    }
-
-    pub mod labels {
-        pub mod v1 {
-            include!(concat!(env!("OUT_DIR"), "/discord_smite_rpc.labels.v1.rs"));
+pub mod osprey {
+    pub mod rpc {
+        pub mod actions {
+            pub mod v1 {
+                include!(concat!(env!("OUT_DIR"), "/osprey.rpc.actions.v1.rs"));
+            }
         }
-    }
-
-    pub mod smite_coordinator {
-        pub mod bidirectional_stream {
+        pub mod common {
             pub mod v1 {
-                include!(concat!(
-                    env!("OUT_DIR"),
-                    "/discord_smite_rpc.smite_coordinator.bidirectional_stream.v1.rs"
-                ));
+                include!(concat!(env!("OUT_DIR"), "/osprey.rpc.common.v1.rs"));
             }
         }
 
-        pub mod sync_action {
+        pub mod labels {
```

Comment:
This will be removed when I remove the label service calls in `sync_action.rs`. I don't want to touch the code logic in the same PR.